WARNING:pygls.protocol:Cancel notification for unknown message id "2"
WARNING:pygls.protocol:Cancel notification for unknown message id "8"
WARNING:pygls.protocol:Cancel notification for unknown message id "14"
WARNING:pygls.protocol:Cancel notification for unknown message id "17"
WARNING:pygls.protocol:Cancel notification for unknown message id "18"
WARNING:pygls.protocol:Cancel notification for unknown message id "34"
WARNING:pygls.protocol:Cancel notification for unknown message id "35"
WARNING:pygls.protocol:Cancel notification for unknown message id "43"
WARNING:pygls.protocol:Cancel notification for unknown message id "58"
WARNING:pygls.protocol:Cancel notification for unknown message id "63"
WARNING:pygls.protocol:Cancel notification for unknown message id "64"
WARNING:pygls.protocol:Cancel notification for unknown message id "69"
ERROR:pygls.protocol:Failed to handle request 70 textDocument/hover work_done_token=None text_document=TextDocumentIdentifier(uri='file:///home/ubuntu/buenComensalProject/restaurants/api/serializers.py') position=41:0
Traceback (most recent call last):
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request
    self._execute_request(msg_id, handler, params)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request
    method_name, method_type, msg_id, handler(params))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 352, in hover
    jedi_script.help(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 484, in wrapper
    raise ValueError('`column` parameter (%d) is not in a valid range '
ValueError: `column` parameter (1) is not in a valid range (0-0) for line 42 ('\n').
[Error - 1:45:25 AM] Request textDocument/hover failed.
  Message: ValueError: `column` parameter (1) is not in a valid range (0-0) for line 42 ('\n').
  Code: -32602 
{'traceback': ['  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request\n    self._execute_request(msg_id, handler, params)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request\n    method_name, method_type, msg_id, handler(params))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 352, in hover\n    jedi_script.help(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 484, in wrapper\n    raise ValueError(\'`column` parameter (%d) is not in a valid range \'\n']}
WARNING:pygls.protocol:Cancel notification for unknown message id "80"
WARNING:pygls.protocol:Cancel notification for unknown message id "81"
WARNING:pygls.protocol:Cancel notification for unknown message id "83"
WARNING:pygls.protocol:Cancel notification for unknown message id "93"
WARNING:pygls.protocol:Cancel notification for unknown message id "94"
WARNING:pygls.protocol:Cancel notification for unknown message id "96"
WARNING:pygls.protocol:Cancel notification for unknown message id "98"
WARNING:pygls.protocol:Cancel notification for unknown message id "99"
WARNING:pygls.protocol:Cancel notification for unknown message id "100"
WARNING:pygls.protocol:Cancel notification for unknown message id "102"
WARNING:pygls.protocol:Cancel notification for unknown message id "103"
WARNING:pygls.protocol:Cancel notification for unknown message id "104"
WARNING:pygls.protocol:Cancel notification for unknown message id "106"
WARNING:pygls.protocol:Cancel notification for unknown message id "108"
WARNING:pygls.protocol:Cancel notification for unknown message id "109"
WARNING:pygls.protocol:Cancel notification for unknown message id "111"
WARNING:pygls.protocol:Cancel notification for unknown message id "116"
WARNING:pygls.protocol:Cancel notification for unknown message id "119"
WARNING:pygls.protocol:Cancel notification for unknown message id "118"
WARNING:pygls.protocol:Cancel notification for unknown message id "120"
WARNING:pygls.protocol:Cancel notification for unknown message id "122"
WARNING:pygls.protocol:Cancel notification for unknown message id "125"
WARNING:pygls.protocol:Cancel notification for unknown message id "123"
WARNING:pygls.protocol:Cancel notification for unknown message id "126"
WARNING:pygls.protocol:Cancel notification for unknown message id "127"
WARNING:pygls.protocol:Cancel notification for unknown message id "141"
WARNING:pygls.protocol:Cancel notification for unknown message id "145"
WARNING:pygls.protocol:Cancel notification for unknown message id "151"
WARNING:pygls.protocol:Cancel notification for unknown message id "153"
WARNING:pygls.protocol:Cancel notification for unknown message id "159"
WARNING:pygls.protocol:Cancel notification for unknown message id "161"
WARNING:pygls.protocol:Cancel notification for unknown message id "170"
WARNING:pygls.protocol:Cancel notification for unknown message id "169"
WARNING:pygls.protocol:Cancel notification for unknown message id "173"
WARNING:pygls.protocol:Cancel notification for unknown message id "182"
WARNING:pygls.protocol:Cancel notification for unknown message id "186"
WARNING:pygls.protocol:Cancel notification for unknown message id "192"
WARNING:pygls.protocol:Cancel notification for unknown message id "199"
WARNING:pygls.protocol:Cancel notification for unknown message id "201"
WARNING:pygls.protocol:Cancel notification for unknown message id "204"
WARNING:pygls.protocol:Cancel notification for unknown message id "202"
WARNING:pygls.protocol:Cancel notification for unknown message id "206"
WARNING:pygls.protocol:Cancel notification for unknown message id "208"
WARNING:pygls.protocol:Cancel notification for unknown message id "211"
WARNING:pygls.protocol:Cancel notification for unknown message id "216"
WARNING:pygls.protocol:Cancel notification for unknown message id "239"
WARNING:pygls.protocol:Cancel notification for unknown message id "241"
WARNING:pygls.protocol:Cancel notification for unknown message id "243"
WARNING:pygls.protocol:Cancel notification for unknown message id "246"
WARNING:pygls.protocol:Cancel notification for unknown message id "245"
WARNING:pygls.protocol:Cancel notification for unknown message id "242"
WARNING:pygls.protocol:Cancel notification for unknown message id "247"
WARNING:pygls.protocol:Cancel notification for unknown message id "248"
WARNING:pygls.protocol:Cancel notification for unknown message id "250"
WARNING:pygls.protocol:Cancel notification for unknown message id "252"
WARNING:pygls.protocol:Cancel notification for unknown message id "251"
WARNING:pygls.protocol:Cancel notification for unknown message id "253"
WARNING:pygls.protocol:Cancel notification for unknown message id "254"
WARNING:pygls.protocol:Cancel notification for unknown message id "256"
WARNING:pygls.protocol:Cancel notification for unknown message id "255"
WARNING:pygls.protocol:Cancel notification for unknown message id "259"
WARNING:pygls.protocol:Cancel notification for unknown message id "257"
WARNING:pygls.protocol:Cancel notification for unknown message id "261"
WARNING:pygls.protocol:Cancel notification for unknown message id "258"
WARNING:pygls.protocol:Cancel notification for unknown message id "260"
WARNING:pygls.protocol:Cancel notification for unknown message id "262"
WARNING:pygls.protocol:Cancel notification for unknown message id "263"
WARNING:pygls.protocol:Cancel notification for unknown message id "264"
WARNING:pygls.protocol:Cancel notification for unknown message id "265"
WARNING:pygls.protocol:Cancel notification for unknown message id "266"
WARNING:pygls.protocol:Cancel notification for unknown message id "267"
WARNING:pygls.protocol:Cancel notification for unknown message id "268"
WARNING:pygls.protocol:Cancel notification for unknown message id "269"
WARNING:pygls.protocol:Cancel notification for unknown message id "270"
WARNING:pygls.protocol:Cancel notification for unknown message id "273"
WARNING:pygls.protocol:Cancel notification for unknown message id "272"
WARNING:pygls.protocol:Cancel notification for unknown message id "274"
WARNING:pygls.protocol:Cancel notification for unknown message id "275"
WARNING:pygls.protocol:Cancel notification for unknown message id "271"
WARNING:pygls.protocol:Cancel notification for unknown message id "276"
WARNING:pygls.protocol:Cancel notification for unknown message id "279"
WARNING:pygls.protocol:Cancel notification for unknown message id "278"
WARNING:pygls.protocol:Cancel notification for unknown message id "282"
WARNING:pygls.protocol:Cancel notification for unknown message id "281"
WARNING:pygls.protocol:Cancel notification for unknown message id "283"
WARNING:pygls.protocol:Cancel notification for unknown message id "284"
WARNING:pygls.protocol:Cancel notification for unknown message id "285"
WARNING:pygls.protocol:Cancel notification for unknown message id "287"
WARNING:pygls.protocol:Cancel notification for unknown message id "280"
WARNING:pygls.protocol:Cancel notification for unknown message id "288"
WARNING:pygls.protocol:Cancel notification for unknown message id "290"
WARNING:pygls.protocol:Cancel notification for unknown message id "292"
WARNING:pygls.protocol:Cancel notification for unknown message id "291"
WARNING:pygls.protocol:Cancel notification for unknown message id "294"
WARNING:pygls.protocol:Cancel notification for unknown message id "293"
WARNING:pygls.protocol:Cancel notification for unknown message id "286"
WARNING:pygls.protocol:Cancel notification for unknown message id "296"
ERROR:pygls.protocol:Failed to handle request 297 textDocument/documentHighlight partial_result_token=None work_done_token=None text_document=TextDocumentIdentifier(uri='file:///home/ubuntu/buenComensalProject/restaurants/api/serializers.py') position=49:12
Traceback (most recent call last):
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request
    self._execute_request(msg_id, handler, params)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request
    method_name, method_type, msg_id, handler(params))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 329, in highlight
    names = jedi_script.get_references(*jedi_lines, scope="file")
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper
    return func(self, line, column, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 418, in get_references
    return _references(**kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 412, in _references
    names = find_references(self._get_module_context(), tree_name, scope == 'file')
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/references.py", line 147, in find_references
    new = _dictionarize(_find_names(module_context, name_leaf))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/references.py", line 74, in _find_names
    found_names = set(name.goto())
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/names.py", line 182, in goto
    value_set = context.infer_node(to_infer[0])
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/context.py", line 224, in infer_node
    return infer_node(self, node)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 156, in infer_node
    return _infer_node_if_inferred(context, element)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 169, in _infer_node_if_inferred
    return _infer_node_cached(context, element)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/cache.py", line 44, in wrapper
    rv = function(obj, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 174, in _infer_node_cached
    return _infer_node(context, element)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/debug.py", line 81, in wrapper
    return func(*args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 82, in wrapper
    return func(context, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 254, in _infer_node
    return infer_or_test(context, element)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 459, in infer_or_test
    iterator = iter(or_test.children)
AttributeError: 'Newline' object has no attribute 'children'
[Error - 2:07:10 AM] Request textDocument/documentHighlight failed.
  Message: AttributeError: 'Newline' object has no attribute 'children'
  Code: -32602 
{'traceback': ['  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request\n    self._execute_request(msg_id, handler, params)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request\n    method_name, method_type, msg_id, handler(params))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 329, in highlight\n    names = jedi_script.get_references(*jedi_lines, scope="file")\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper\n    return func(self, line, column, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 418, in get_references\n    return _references(**kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 412, in _references\n    names = find_references(self._get_module_context(), tree_name, scope == \'file\')\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/references.py", line 147, in find_references\n    new = _dictionarize(_find_names(module_context, name_leaf))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/references.py", line 74, in _find_names\n    found_names = set(name.goto())\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/names.py", line 182, in goto\n    value_set = context.infer_node(to_infer[0])\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/context.py", line 224, in infer_node\n    return infer_node(self, node)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 156, in infer_node\n    return _infer_node_if_inferred(context, element)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 169, in _infer_node_if_inferred\n    return _infer_node_cached(context, element)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/cache.py", line 44, in wrapper\n    rv = function(obj, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 174, in _infer_node_cached\n    return _infer_node(context, element)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/debug.py", line 81, in wrapper\n    return func(*args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 82, in wrapper\n    return func(context, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 254, in _infer_node\n    return infer_or_test(context, element)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/inference/syntax_tree.py", line 459, in infer_or_test\n    iterator = iter(or_test.children)\n']}
WARNING:pygls.protocol:Cancel notification for unknown message id "295"
WARNING:pygls.protocol:Cancel notification for unknown message id "297"
WARNING:pygls.protocol:Cancel notification for unknown message id "299"
WARNING:pygls.protocol:Cancel notification for unknown message id "298"
WARNING:pygls.protocol:Cancel notification for unknown message id "300"
WARNING:pygls.protocol:Cancel notification for unknown message id "301"
WARNING:pygls.protocol:Cancel notification for unknown message id "303"
WARNING:pygls.protocol:Cancel notification for unknown message id "302"
WARNING:pygls.protocol:Cancel notification for unknown message id "304"
WARNING:pygls.protocol:Cancel notification for unknown message id "305"
WARNING:pygls.protocol:Cancel notification for unknown message id "306"
WARNING:pygls.protocol:Cancel notification for unknown message id "307"
WARNING:pygls.protocol:Cancel notification for unknown message id "308"
WARNING:pygls.protocol:Cancel notification for unknown message id "309"
WARNING:pygls.protocol:Cancel notification for unknown message id "310"
WARNING:pygls.protocol:Cancel notification for unknown message id "311"
WARNING:pygls.protocol:Cancel notification for unknown message id "314"
WARNING:pygls.protocol:Cancel notification for unknown message id "315"
WARNING:pygls.protocol:Cancel notification for unknown message id "313"
WARNING:pygls.protocol:Cancel notification for unknown message id "316"
WARNING:pygls.protocol:Cancel notification for unknown message id "317"
WARNING:pygls.protocol:Cancel notification for unknown message id "318"
WARNING:pygls.protocol:Cancel notification for unknown message id "319"
WARNING:pygls.protocol:Cancel notification for unknown message id "320"
WARNING:pygls.protocol:Cancel notification for unknown message id "321"
WARNING:pygls.protocol:Cancel notification for unknown message id "325"
WARNING:pygls.protocol:Cancel notification for unknown message id "331"
WARNING:pygls.protocol:Cancel notification for unknown message id "349"
WARNING:pygls.protocol:Cancel notification for unknown message id "363"
WARNING:pygls.protocol:Cancel notification for unknown message id "370"
WARNING:pygls.protocol:Cancel notification for unknown message id "372"
WARNING:pygls.protocol:Cancel notification for unknown message id "375"
WARNING:pygls.protocol:Cancel notification for unknown message id "376"
WARNING:pygls.protocol:Cancel notification for unknown message id "379"
WARNING:pygls.protocol:Cancel notification for unknown message id "382"
WARNING:pygls.protocol:Cancel notification for unknown message id "401"
WARNING:pygls.protocol:Cancel notification for unknown message id "402"
WARNING:pygls.protocol:Cancel notification for unknown message id "404"
WARNING:pygls.protocol:Cancel notification for unknown message id "408"
WARNING:pygls.protocol:Cancel notification for unknown message id "410"
WARNING:pygls.protocol:Cancel notification for unknown message id "412"
WARNING:pygls.protocol:Cancel notification for unknown message id "411"
WARNING:pygls.protocol:Cancel notification for unknown message id "414"
WARNING:pygls.protocol:Cancel notification for unknown message id "416"
WARNING:pygls.protocol:Cancel notification for unknown message id "417"
WARNING:pygls.protocol:Cancel notification for unknown message id "420"
WARNING:pygls.protocol:Cancel notification for unknown message id "430"
WARNING:pygls.protocol:Cancel notification for unknown message id "431"
WARNING:pygls.protocol:Cancel notification for unknown message id "432"
WARNING:pygls.protocol:Cancel notification for unknown message id "433"
WARNING:pygls.protocol:Cancel notification for unknown message id "434"
WARNING:pygls.protocol:Cancel notification for unknown message id "439"
WARNING:pygls.protocol:Cancel notification for unknown message id "441"
WARNING:pygls.protocol:Cancel notification for unknown message id "443"
WARNING:pygls.protocol:Cancel notification for unknown message id "444"
WARNING:pygls.protocol:Cancel notification for unknown message id "445"
WARNING:pygls.protocol:Cancel notification for unknown message id "446"
WARNING:pygls.protocol:Cancel notification for unknown message id "447"
WARNING:pygls.protocol:Cancel notification for unknown message id "449"
WARNING:pygls.protocol:Cancel notification for unknown message id "453"
WARNING:pygls.protocol:Cancel notification for unknown message id "456"
WARNING:pygls.protocol:Cancel notification for unknown message id "455"
WARNING:pygls.protocol:Cancel notification for unknown message id "457"
WARNING:pygls.protocol:Cancel notification for unknown message id "459"
WARNING:pygls.protocol:Cancel notification for unknown message id "460"
WARNING:pygls.protocol:Cancel notification for unknown message id "465"
WARNING:pygls.protocol:Cancel notification for unknown message id "472"
WARNING:pygls.protocol:Cancel notification for unknown message id "473"
WARNING:pygls.protocol:Cancel notification for unknown message id "476"
WARNING:pygls.protocol:Cancel notification for unknown message id "484"
WARNING:pygls.protocol:Cancel notification for unknown message id "488"
WARNING:pygls.protocol:Cancel notification for unknown message id "495"
WARNING:pygls.protocol:Cancel notification for unknown message id "496"
WARNING:pygls.protocol:Cancel notification for unknown message id "497"
WARNING:pygls.protocol:Cancel notification for unknown message id "498"
WARNING:pygls.protocol:Cancel notification for unknown message id "501"
WARNING:pygls.protocol:Cancel notification for unknown message id "507"
WARNING:pygls.protocol:Cancel notification for unknown message id "506"
WARNING:pygls.protocol:Cancel notification for unknown message id "509"
WARNING:pygls.protocol:Cancel notification for unknown message id "510"
WARNING:pygls.protocol:Cancel notification for unknown message id "511"
WARNING:pygls.protocol:Cancel notification for unknown message id "572"
WARNING:pygls.protocol:Cancel notification for unknown message id "573"
WARNING:pygls.protocol:Cancel notification for unknown message id "576"
WARNING:pygls.protocol:Cancel notification for unknown message id "577"
WARNING:pygls.protocol:Cancel notification for unknown message id "592"
WARNING:pygls.protocol:Cancel notification for unknown message id "593"
WARNING:pygls.protocol:Cancel notification for unknown message id "594"
WARNING:pygls.protocol:Cancel notification for unknown message id "595"
WARNING:pygls.protocol:Cancel notification for unknown message id "596"
WARNING:pygls.protocol:Cancel notification for unknown message id "597"
WARNING:pygls.protocol:Cancel notification for unknown message id "598"
WARNING:pygls.protocol:Cancel notification for unknown message id "599"
WARNING:pygls.protocol:Cancel notification for unknown message id "600"
WARNING:pygls.protocol:Cancel notification for unknown message id "601"
WARNING:pygls.protocol:Cancel notification for unknown message id "624"
WARNING:pygls.protocol:Cancel notification for unknown message id "627"
WARNING:pygls.protocol:Cancel notification for unknown message id "626"
WARNING:pygls.protocol:Cancel notification for unknown message id "628"
WARNING:pygls.protocol:Cancel notification for unknown message id "693"
WARNING:pygls.protocol:Cancel notification for unknown message id "694"
WARNING:pygls.protocol:Cancel notification for unknown message id "698"
WARNING:pygls.protocol:Cancel notification for unknown message id "724"
WARNING:pygls.protocol:Cancel notification for unknown message id "729"
WARNING:pygls.protocol:Cancel notification for unknown message id "732"
WARNING:pygls.protocol:Cancel notification for unknown message id "751"
WARNING:pygls.protocol:Cancel notification for unknown message id "777"
WARNING:pygls.protocol:Cancel notification for unknown message id "802"
WARNING:pygls.protocol:Cancel notification for unknown message id "803"
WARNING:pygls.protocol:Cancel notification for unknown message id "809"
WARNING:pygls.protocol:Cancel notification for unknown message id "817"
WARNING:pygls.protocol:Cancel notification for unknown message id "836"
WARNING:pygls.protocol:Cancel notification for unknown message id "850"
WARNING:pygls.protocol:Cancel notification for unknown message id "872"
WARNING:pygls.protocol:Cancel notification for unknown message id "874"
WARNING:pygls.protocol:Cancel notification for unknown message id "882"
WARNING:pygls.protocol:Cancel notification for unknown message id "883"
WARNING:pygls.protocol:Cancel notification for unknown message id "885"
WARNING:pygls.protocol:Cancel notification for unknown message id "894"
WARNING:pygls.protocol:Cancel notification for unknown message id "895"
WARNING:pygls.protocol:Cancel notification for unknown message id "897"
WARNING:pygls.protocol:Cancel notification for unknown message id "896"
WARNING:pygls.protocol:Cancel notification for unknown message id "899"
WARNING:pygls.protocol:Cancel notification for unknown message id "898"
WARNING:pygls.protocol:Cancel notification for unknown message id "902"
WARNING:pygls.protocol:Cancel notification for unknown message id "901"
WARNING:pygls.protocol:Cancel notification for unknown message id "900"
WARNING:pygls.protocol:Cancel notification for unknown message id "905"
WARNING:pygls.protocol:Cancel notification for unknown message id "904"
WARNING:pygls.protocol:Cancel notification for unknown message id "908"
WARNING:pygls.protocol:Cancel notification for unknown message id "907"
WARNING:pygls.protocol:Cancel notification for unknown message id "909"
WARNING:pygls.protocol:Cancel notification for unknown message id "912"
WARNING:pygls.protocol:Cancel notification for unknown message id "906"
WARNING:pygls.protocol:Cancel notification for unknown message id "911"
WARNING:pygls.protocol:Cancel notification for unknown message id "916"
WARNING:pygls.protocol:Cancel notification for unknown message id "910"
WARNING:pygls.protocol:Cancel notification for unknown message id "915"
WARNING:pygls.protocol:Cancel notification for unknown message id "914"
WARNING:pygls.protocol:Cancel notification for unknown message id "917"
WARNING:pygls.protocol:Cancel notification for unknown message id "918"
WARNING:pygls.protocol:Cancel notification for unknown message id "920"
WARNING:pygls.protocol:Cancel notification for unknown message id "919"
WARNING:pygls.protocol:Cancel notification for unknown message id "922"
WARNING:pygls.protocol:Cancel notification for unknown message id "923"
WARNING:pygls.protocol:Cancel notification for unknown message id "924"
WARNING:pygls.protocol:Cancel notification for unknown message id "960"
WARNING:pygls.protocol:Cancel notification for unknown message id "988"
WARNING:pygls.protocol:Cancel notification for unknown message id "998"
WARNING:pygls.protocol:Cancel notification for unknown message id "1002"
WARNING:pygls.protocol:Cancel notification for unknown message id "1013"
WARNING:pygls.protocol:Cancel notification for unknown message id "1028"
WARNING:pygls.protocol:Cancel notification for unknown message id "1030"
WARNING:pygls.protocol:Cancel notification for unknown message id "1029"
WARNING:pygls.protocol:Cancel notification for unknown message id "1058"
WARNING:pygls.protocol:Cancel notification for unknown message id "1059"
WARNING:pygls.protocol:Cancel notification for unknown message id "1060"
WARNING:pygls.protocol:Cancel notification for unknown message id "1061"
WARNING:pygls.protocol:Cancel notification for unknown message id "1064"
WARNING:pygls.protocol:Cancel notification for unknown message id "1063"
WARNING:pygls.protocol:Cancel notification for unknown message id "1065"
WARNING:pygls.protocol:Cancel notification for unknown message id "1067"
WARNING:pygls.protocol:Cancel notification for unknown message id "1073"
WARNING:pygls.protocol:Cancel notification for unknown message id "1081"
WARNING:pygls.protocol:Cancel notification for unknown message id "1082"
WARNING:pygls.protocol:Cancel notification for unknown message id "1083"
WARNING:pygls.protocol:Cancel notification for unknown message id "1087"
WARNING:pygls.protocol:Cancel notification for unknown message id "1090"
WARNING:pygls.protocol:Cancel notification for unknown message id "1089"
WARNING:pygls.protocol:Cancel notification for unknown message id "1091"
WARNING:pygls.protocol:Cancel notification for unknown message id "1093"
WARNING:pygls.protocol:Cancel notification for unknown message id "1094"
WARNING:pygls.protocol:Cancel notification for unknown message id "1095"
WARNING:pygls.protocol:Cancel notification for unknown message id "1100"
WARNING:pygls.protocol:Cancel notification for unknown message id "1096"
WARNING:pygls.protocol:Cancel notification for unknown message id "1098"
WARNING:pygls.protocol:Cancel notification for unknown message id "1102"
WARNING:pygls.protocol:Cancel notification for unknown message id "1101"
WARNING:pygls.protocol:Cancel notification for unknown message id "1104"
WARNING:pygls.protocol:Cancel notification for unknown message id "1103"
WARNING:pygls.protocol:Cancel notification for unknown message id "1106"
WARNING:pygls.protocol:Cancel notification for unknown message id "1105"
WARNING:pygls.protocol:Cancel notification for unknown message id "1107"
WARNING:pygls.protocol:Cancel notification for unknown message id "1109"
WARNING:pygls.protocol:Cancel notification for unknown message id "1114"
WARNING:pygls.protocol:Cancel notification for unknown message id "1113"
WARNING:pygls.protocol:Cancel notification for unknown message id "1115"
WARNING:pygls.protocol:Cancel notification for unknown message id "1125"
WARNING:pygls.protocol:Cancel notification for unknown message id "1126"
WARNING:pygls.protocol:Cancel notification for unknown message id "1127"
WARNING:pygls.protocol:Cancel notification for unknown message id "1129"
WARNING:pygls.protocol:Cancel notification for unknown message id "1128"
WARNING:pygls.protocol:Cancel notification for unknown message id "1130"
WARNING:pygls.protocol:Cancel notification for unknown message id "1135"
WARNING:pygls.protocol:Cancel notification for unknown message id "1137"
WARNING:pygls.protocol:Cancel notification for unknown message id "1150"
WARNING:pygls.protocol:Cancel notification for unknown message id "1161"
WARNING:pygls.protocol:Cancel notification for unknown message id "1166"
WARNING:pygls.protocol:Cancel notification for unknown message id "1174"
WARNING:pygls.protocol:Cancel notification for unknown message id "1191"
WARNING:pygls.protocol:Cancel notification for unknown message id "1212"
WARNING:pygls.protocol:Cancel notification for unknown message id "1240"
WARNING:pygls.protocol:Cancel notification for unknown message id "1258"
WARNING:pygls.protocol:Cancel notification for unknown message id "1261"
WARNING:pygls.protocol:Cancel notification for unknown message id "1262"
WARNING:pygls.protocol:Cancel notification for unknown message id "1265"
WARNING:pygls.protocol:Cancel notification for unknown message id "1267"
WARNING:pygls.protocol:Cancel notification for unknown message id "1275"
WARNING:pygls.protocol:Cancel notification for unknown message id "1277"
WARNING:pygls.protocol:Cancel notification for unknown message id "1330"
WARNING:pygls.protocol:Cancel notification for unknown message id "1370"
WARNING:pygls.protocol:Cancel notification for unknown message id "1386"
WARNING:pygls.protocol:Cancel notification for unknown message id "1397"
WARNING:pygls.protocol:Cancel notification for unknown message id "1409"
WARNING:pygls.protocol:Cancel notification for unknown message id "1411"
WARNING:pygls.protocol:Cancel notification for unknown message id "1410"
WARNING:pygls.protocol:Cancel notification for unknown message id "1413"
WARNING:pygls.protocol:Cancel notification for unknown message id "1412"
WARNING:pygls.protocol:Cancel notification for unknown message id "1415"
WARNING:pygls.protocol:Cancel notification for unknown message id "1416"
WARNING:pygls.protocol:Cancel notification for unknown message id "1422"
WARNING:pygls.protocol:Cancel notification for unknown message id "1425"
WARNING:pygls.protocol:Cancel notification for unknown message id "1453"
WARNING:pygls.protocol:Cancel notification for unknown message id "1462"
WARNING:pygls.protocol:Cancel notification for unknown message id "1467"
WARNING:pygls.protocol:Cancel notification for unknown message id "1468"
WARNING:pygls.protocol:Cancel notification for unknown message id "1471"
WARNING:pygls.protocol:Cancel notification for unknown message id "1470"
WARNING:pygls.protocol:Cancel notification for unknown message id "1472"
WARNING:pygls.protocol:Cancel notification for unknown message id "1473"
WARNING:pygls.protocol:Cancel notification for unknown message id "1475"
WARNING:pygls.protocol:Cancel notification for unknown message id "1474"
WARNING:pygls.protocol:Cancel notification for unknown message id "1476"
WARNING:pygls.protocol:Cancel notification for unknown message id "1479"
WARNING:pygls.protocol:Cancel notification for unknown message id "1481"
WARNING:pygls.protocol:Cancel notification for unknown message id "1480"
WARNING:pygls.protocol:Cancel notification for unknown message id "1486"
WARNING:pygls.protocol:Cancel notification for unknown message id "1487"
WARNING:pygls.protocol:Cancel notification for unknown message id "1488"
WARNING:pygls.protocol:Cancel notification for unknown message id "1497"
WARNING:pygls.protocol:Cancel notification for unknown message id "1538"
WARNING:pygls.protocol:Cancel notification for unknown message id "1539"
WARNING:pygls.protocol:Cancel notification for unknown message id "1536"
WARNING:pygls.protocol:Cancel notification for unknown message id "1541"
WARNING:pygls.protocol:Cancel notification for unknown message id "1537"
WARNING:pygls.protocol:Cancel notification for unknown message id "1544"
WARNING:pygls.protocol:Cancel notification for unknown message id "1545"
WARNING:pygls.protocol:Cancel notification for unknown message id "1542"
WARNING:pygls.protocol:Cancel notification for unknown message id "1543"
WARNING:pygls.protocol:Cancel notification for unknown message id "1546"
WARNING:pygls.protocol:Cancel notification for unknown message id "1549"
WARNING:pygls.protocol:Cancel notification for unknown message id "1552"
WARNING:pygls.protocol:Cancel notification for unknown message id "1553"
WARNING:pygls.protocol:Cancel notification for unknown message id "1557"
WARNING:pygls.protocol:Cancel notification for unknown message id "1558"
WARNING:pygls.protocol:Cancel notification for unknown message id "1565"
WARNING:pygls.protocol:Cancel notification for unknown message id "1635"
WARNING:pygls.protocol:Cancel notification for unknown message id "1639"
WARNING:pygls.protocol:Cancel notification for unknown message id "1679"
WARNING:pygls.protocol:Cancel notification for unknown message id "1699"
WARNING:pygls.protocol:Cancel notification for unknown message id "1745"
WARNING:pygls.protocol:Cancel notification for unknown message id "1754"
WARNING:pygls.protocol:Cancel notification for unknown message id "1858"
WARNING:pygls.protocol:Cancel notification for unknown message id "1861"
WARNING:pygls.protocol:Cancel notification for unknown message id "1863"
WARNING:pygls.protocol:Cancel notification for unknown message id "1870"
WARNING:pygls.protocol:Cancel notification for unknown message id "1874"
WARNING:pygls.protocol:Cancel notification for unknown message id "1879"
WARNING:pygls.protocol:Cancel notification for unknown message id "1880"
WARNING:pygls.protocol:Cancel notification for unknown message id "1882"
WARNING:pygls.protocol:Cancel notification for unknown message id "1883"
WARNING:pygls.protocol:Cancel notification for unknown message id "1885"
WARNING:pygls.protocol:Cancel notification for unknown message id "1890"
WARNING:pygls.protocol:Cancel notification for unknown message id "1891"
WARNING:pygls.protocol:Cancel notification for unknown message id "1892"
WARNING:pygls.protocol:Cancel notification for unknown message id "1901"
WARNING:pygls.protocol:Cancel notification for unknown message id "1904"
WARNING:pygls.protocol:Cancel notification for unknown message id "1906"
WARNING:pygls.protocol:Cancel notification for unknown message id "1908"
WARNING:pygls.protocol:Cancel notification for unknown message id "1910"
WARNING:pygls.protocol:Cancel notification for unknown message id "1912"
WARNING:pygls.protocol:Cancel notification for unknown message id "1911"
WARNING:pygls.protocol:Cancel notification for unknown message id "1923"
WARNING:pygls.protocol:Cancel notification for unknown message id "1924"
WARNING:pygls.protocol:Cancel notification for unknown message id "1925"
WARNING:pygls.protocol:Cancel notification for unknown message id "1933"
WARNING:pygls.protocol:Cancel notification for unknown message id "1949"
WARNING:pygls.protocol:Cancel notification for unknown message id "1952"
WARNING:pygls.protocol:Cancel notification for unknown message id "1960"
WARNING:pygls.protocol:Cancel notification for unknown message id "1961"
WARNING:pygls.protocol:Cancel notification for unknown message id "1962"
WARNING:pygls.protocol:Cancel notification for unknown message id "1967"
WARNING:pygls.protocol:Cancel notification for unknown message id "1985"
WARNING:pygls.protocol:Cancel notification for unknown message id "1989"
WARNING:pygls.protocol:Cancel notification for unknown message id "1994"
WARNING:pygls.protocol:Cancel notification for unknown message id "1995"
WARNING:pygls.protocol:Cancel notification for unknown message id "2012"
WARNING:pygls.protocol:Cancel notification for unknown message id "2014"
WARNING:pygls.protocol:Cancel notification for unknown message id "2018"
WARNING:pygls.protocol:Cancel notification for unknown message id "2019"
WARNING:pygls.protocol:Cancel notification for unknown message id "2034"
WARNING:pygls.protocol:Cancel notification for unknown message id "2037"
WARNING:pygls.protocol:Cancel notification for unknown message id "2038"
WARNING:pygls.protocol:Cancel notification for unknown message id "2042"
WARNING:pygls.protocol:Cancel notification for unknown message id "2051"
WARNING:pygls.protocol:Cancel notification for unknown message id "2053"
WARNING:pygls.protocol:Cancel notification for unknown message id "2058"
WARNING:pygls.protocol:Cancel notification for unknown message id "2068"
WARNING:pygls.protocol:Cancel notification for unknown message id "2107"
WARNING:pygls.protocol:Cancel notification for unknown message id "2124"
WARNING:pygls.protocol:Cancel notification for unknown message id "2131"
WARNING:pygls.protocol:Cancel notification for unknown message id "2132"
WARNING:pygls.protocol:Cancel notification for unknown message id "2133"
WARNING:pygls.protocol:Cancel notification for unknown message id "2134"
WARNING:pygls.protocol:Cancel notification for unknown message id "2136"
WARNING:pygls.protocol:Cancel notification for unknown message id "2138"
WARNING:pygls.protocol:Cancel notification for unknown message id "2152"
WARNING:pygls.protocol:Cancel notification for unknown message id "2160"
WARNING:pygls.protocol:Cancel notification for unknown message id "2162"
WARNING:pygls.protocol:Cancel notification for unknown message id "2163"
WARNING:pygls.protocol:Cancel notification for unknown message id "2164"
WARNING:pygls.protocol:Cancel notification for unknown message id "2165"
WARNING:pygls.protocol:Cancel notification for unknown message id "2167"
WARNING:pygls.protocol:Cancel notification for unknown message id "2166"
WARNING:pygls.protocol:Cancel notification for unknown message id "2171"
WARNING:pygls.protocol:Cancel notification for unknown message id "2173"
WARNING:pygls.protocol:Cancel notification for unknown message id "2181"
WARNING:pygls.protocol:Cancel notification for unknown message id "2185"
WARNING:pygls.protocol:Cancel notification for unknown message id "2186"
WARNING:pygls.protocol:Cancel notification for unknown message id "2193"
WARNING:pygls.protocol:Cancel notification for unknown message id "2195"
WARNING:pygls.protocol:Cancel notification for unknown message id "2196"
WARNING:pygls.protocol:Cancel notification for unknown message id "2197"
WARNING:pygls.protocol:Cancel notification for unknown message id "2199"
WARNING:pygls.protocol:Cancel notification for unknown message id "2206"
WARNING:pygls.protocol:Cancel notification for unknown message id "2205"
WARNING:pygls.protocol:Cancel notification for unknown message id "2222"
WARNING:pygls.protocol:Cancel notification for unknown message id "2223"
WARNING:pygls.protocol:Cancel notification for unknown message id "2224"
WARNING:pygls.protocol:Cancel notification for unknown message id "2229"
WARNING:pygls.protocol:Cancel notification for unknown message id "2228"
WARNING:pygls.protocol:Cancel notification for unknown message id "2231"
WARNING:pygls.protocol:Cancel notification for unknown message id "2233"
WARNING:pygls.protocol:Cancel notification for unknown message id "2235"
WARNING:pygls.protocol:Cancel notification for unknown message id "2234"
WARNING:pygls.protocol:Cancel notification for unknown message id "2237"
WARNING:pygls.protocol:Cancel notification for unknown message id "2239"
WARNING:pygls.protocol:Cancel notification for unknown message id "2240"
WARNING:pygls.protocol:Cancel notification for unknown message id "2241"
WARNING:pygls.protocol:Cancel notification for unknown message id "2242"
WARNING:pygls.protocol:Cancel notification for unknown message id "2255"
WARNING:pygls.protocol:Cancel notification for unknown message id "2256"
WARNING:pygls.protocol:Cancel notification for unknown message id "2258"
WARNING:pygls.protocol:Cancel notification for unknown message id "2269"
WARNING:pygls.protocol:Cancel notification for unknown message id "2271"
WARNING:pygls.protocol:Cancel notification for unknown message id "2272"
WARNING:pygls.protocol:Cancel notification for unknown message id "2274"
WARNING:pygls.protocol:Cancel notification for unknown message id "2275"
WARNING:pygls.protocol:Cancel notification for unknown message id "2276"
WARNING:pygls.protocol:Cancel notification for unknown message id "2278"
WARNING:pygls.protocol:Cancel notification for unknown message id "2279"
WARNING:pygls.protocol:Cancel notification for unknown message id "2277"
WARNING:pygls.protocol:Cancel notification for unknown message id "2280"
WARNING:pygls.protocol:Cancel notification for unknown message id "2281"
WARNING:pygls.protocol:Cancel notification for unknown message id "2283"
WARNING:pygls.protocol:Cancel notification for unknown message id "2285"
WARNING:pygls.protocol:Cancel notification for unknown message id "2286"
WARNING:pygls.protocol:Cancel notification for unknown message id "2287"
WARNING:pygls.protocol:Cancel notification for unknown message id "2288"
WARNING:pygls.protocol:Cancel notification for unknown message id "2313"
WARNING:pygls.protocol:Cancel notification for unknown message id "2329"
WARNING:pygls.protocol:Cancel notification for unknown message id "2333"
WARNING:pygls.protocol:Cancel notification for unknown message id "2359"
WARNING:pygls.protocol:Cancel notification for unknown message id "2361"
WARNING:pygls.protocol:Cancel notification for unknown message id "2372"
WARNING:pygls.protocol:Cancel notification for unknown message id "2373"
WARNING:pygls.protocol:Cancel notification for unknown message id "2382"
WARNING:pygls.protocol:Cancel notification for unknown message id "2384"
WARNING:pygls.protocol:Cancel notification for unknown message id "2385"
WARNING:pygls.protocol:Cancel notification for unknown message id "2388"
WARNING:pygls.protocol:Cancel notification for unknown message id "2389"
WARNING:pygls.protocol:Cancel notification for unknown message id "2393"
WARNING:pygls.protocol:Cancel notification for unknown message id "2392"
WARNING:pygls.protocol:Cancel notification for unknown message id "2395"
WARNING:pygls.protocol:Cancel notification for unknown message id "2394"
WARNING:pygls.protocol:Cancel notification for unknown message id "2396"
WARNING:pygls.protocol:Cancel notification for unknown message id "2397"
WARNING:pygls.protocol:Cancel notification for unknown message id "2399"
WARNING:pygls.protocol:Cancel notification for unknown message id "2401"
WARNING:pygls.protocol:Cancel notification for unknown message id "2403"
WARNING:pygls.protocol:Cancel notification for unknown message id "2404"
WARNING:pygls.protocol:Cancel notification for unknown message id "2405"
WARNING:pygls.protocol:Cancel notification for unknown message id "2408"
WARNING:pygls.protocol:Cancel notification for unknown message id "2407"
WARNING:pygls.protocol:Cancel notification for unknown message id "2411"
WARNING:pygls.protocol:Cancel notification for unknown message id "2410"
WARNING:pygls.protocol:Cancel notification for unknown message id "2409"
WARNING:pygls.protocol:Cancel notification for unknown message id "2412"
WARNING:pygls.protocol:Cancel notification for unknown message id "2414"
WARNING:pygls.protocol:Cancel notification for unknown message id "2415"
WARNING:pygls.protocol:Cancel notification for unknown message id "2417"
WARNING:pygls.protocol:Cancel notification for unknown message id "2419"
WARNING:pygls.protocol:Cancel notification for unknown message id "2418"
WARNING:pygls.protocol:Cancel notification for unknown message id "2421"
WARNING:pygls.protocol:Cancel notification for unknown message id "2424"
WARNING:pygls.protocol:Cancel notification for unknown message id "2433"
WARNING:pygls.protocol:Cancel notification for unknown message id "2438"
WARNING:pygls.protocol:Cancel notification for unknown message id "2549"
WARNING:pygls.protocol:Cancel notification for unknown message id "2559"
ERROR:pygls.protocol:Failed to handle request 2569 textDocument/codeAction partial_result_token=None work_done_token=None text_document=TextDocumentIdentifier(uri='file:///home/ubuntu/buenComensalProject/restaurants/api/api.py') range=19:18-19:18 context=CodeActionContext(diagnostics=[], only=None)
Traceback (most recent call last):
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request
    self._execute_request(msg_id, handler, params)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request
    method_name, method_type, msg_id, handler(params))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action
    extract_variable_refactoring = jedi_script.extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper
    return func(self, line, column, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable
    return extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable
    nodes = _find_nodes(module_node, pos, until_pos)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 68, in _find_nodes
    next_leaf = start_node.get_next_leaf()
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/parso/tree.py", line 121, in get_next_leaf
    i = c.index(node)
ValueError: PythonNode(trailer, [<Operator: (>, PythonNode(atom, [<Operator: [>, <String: 'GET'>, <Operator: ]>]), <Operator: )>]) is not in list
[Error - 3:22:40 AM] Request textDocument/codeAction failed.
  Message: ValueError: PythonNode(trailer, [<Operator: (>, PythonNode(atom, [<Operator: [>, <String: 'GET'>, <Operator: ]>]), <Operator: )>]) is not in list
  Code: -32602 
{'traceback': ['  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request\n    self._execute_request(msg_id, handler, params)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request\n    method_name, method_type, msg_id, handler(params))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action\n    extract_variable_refactoring = jedi_script.extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper\n    return func(self, line, column, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable\n    return extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable\n    nodes = _find_nodes(module_node, pos, until_pos)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 68, in _find_nodes\n    next_leaf = start_node.get_next_leaf()\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/parso/tree.py", line 121, in get_next_leaf\n    i = c.index(node)\n']}
WARNING:pygls.protocol:Cancel notification for unknown message id "2596"
WARNING:pygls.protocol:Cancel notification for unknown message id "2628"
WARNING:pygls.protocol:Cancel notification for unknown message id "2631"
WARNING:pygls.protocol:Cancel notification for unknown message id "2634"
WARNING:pygls.protocol:Cancel notification for unknown message id "2632"
WARNING:pygls.protocol:Cancel notification for unknown message id "2636"
WARNING:pygls.protocol:Cancel notification for unknown message id "2638"
WARNING:pygls.protocol:Cancel notification for unknown message id "2640"
WARNING:pygls.protocol:Cancel notification for unknown message id "2635"
WARNING:pygls.protocol:Cancel notification for unknown message id "2637"
WARNING:pygls.protocol:Cancel notification for unknown message id "2642"
WARNING:pygls.protocol:Cancel notification for unknown message id "2655"
WARNING:pygls.protocol:Cancel notification for unknown message id "2659"
WARNING:pygls.protocol:Cancel notification for unknown message id "2665"
WARNING:pygls.protocol:Cancel notification for unknown message id "2671"
WARNING:pygls.protocol:Cancel notification for unknown message id "2684"
WARNING:pygls.protocol:Cancel notification for unknown message id "2685"
WARNING:pygls.protocol:Cancel notification for unknown message id "2687"
WARNING:pygls.protocol:Cancel notification for unknown message id "2690"
WARNING:pygls.protocol:Cancel notification for unknown message id "2689"
WARNING:pygls.protocol:Cancel notification for unknown message id "2712"
WARNING:pygls.protocol:Cancel notification for unknown message id "2716"
WARNING:pygls.protocol:Cancel notification for unknown message id "2717"
WARNING:pygls.protocol:Cancel notification for unknown message id "2718"
WARNING:pygls.protocol:Cancel notification for unknown message id "2720"
WARNING:pygls.protocol:Cancel notification for unknown message id "2721"
WARNING:pygls.protocol:Cancel notification for unknown message id "2719"
WARNING:pygls.protocol:Cancel notification for unknown message id "2722"
WARNING:pygls.protocol:Cancel notification for unknown message id "2724"
WARNING:pygls.protocol:Cancel notification for unknown message id "2727"
WARNING:pygls.protocol:Cancel notification for unknown message id "2728"
WARNING:pygls.protocol:Cancel notification for unknown message id "2730"
WARNING:pygls.protocol:Cancel notification for unknown message id "2723"
WARNING:pygls.protocol:Cancel notification for unknown message id "2731"
WARNING:pygls.protocol:Cancel notification for unknown message id "2725"
WARNING:pygls.protocol:Cancel notification for unknown message id "2729"
WARNING:pygls.protocol:Cancel notification for unknown message id "2733"
WARNING:pygls.protocol:Cancel notification for unknown message id "2732"
WARNING:pygls.protocol:Cancel notification for unknown message id "2726"
WARNING:pygls.protocol:Cancel notification for unknown message id "2734"
WARNING:pygls.protocol:Cancel notification for unknown message id "2737"
WARNING:pygls.protocol:Cancel notification for unknown message id "2741"
WARNING:pygls.protocol:Cancel notification for unknown message id "2747"
WARNING:pygls.protocol:Cancel notification for unknown message id "2748"
WARNING:pygls.protocol:Cancel notification for unknown message id "2752"
WARNING:pygls.protocol:Cancel notification for unknown message id "2751"
WARNING:pygls.protocol:Cancel notification for unknown message id "2750"
WARNING:pygls.protocol:Cancel notification for unknown message id "2749"
WARNING:pygls.protocol:Cancel notification for unknown message id "2756"
WARNING:pygls.protocol:Cancel notification for unknown message id "2770"
WARNING:pygls.protocol:Cancel notification for unknown message id "2772"
WARNING:pygls.protocol:Cancel notification for unknown message id "2773"
WARNING:pygls.protocol:Cancel notification for unknown message id "2778"
WARNING:pygls.protocol:Cancel notification for unknown message id "2779"
WARNING:pygls.protocol:Cancel notification for unknown message id "2788"
WARNING:pygls.protocol:Cancel notification for unknown message id "2792"
WARNING:pygls.protocol:Cancel notification for unknown message id "2793"
WARNING:pygls.protocol:Cancel notification for unknown message id "2820"
WARNING:pygls.protocol:Cancel notification for unknown message id "2824"
WARNING:pygls.protocol:Cancel notification for unknown message id "2840"
WARNING:pygls.protocol:Cancel notification for unknown message id "2841"
WARNING:pygls.protocol:Cancel notification for unknown message id "2846"
WARNING:pygls.protocol:Cancel notification for unknown message id "2857"
WARNING:pygls.protocol:Cancel notification for unknown message id "2859"
WARNING:pygls.protocol:Cancel notification for unknown message id "2861"
WARNING:pygls.protocol:Cancel notification for unknown message id "2860"
WARNING:pygls.protocol:Cancel notification for unknown message id "2864"
WARNING:pygls.protocol:Cancel notification for unknown message id "2863"
WARNING:pygls.protocol:Cancel notification for unknown message id "2862"
WARNING:pygls.protocol:Cancel notification for unknown message id "2868"
WARNING:pygls.protocol:Cancel notification for unknown message id "2869"
WARNING:pygls.protocol:Cancel notification for unknown message id "2865"
WARNING:pygls.protocol:Cancel notification for unknown message id "2866"
WARNING:pygls.protocol:Cancel notification for unknown message id "2870"
WARNING:pygls.protocol:Cancel notification for unknown message id "2871"
WARNING:pygls.protocol:Cancel notification for unknown message id "2867"
WARNING:pygls.protocol:Cancel notification for unknown message id "2879"
WARNING:pygls.protocol:Cancel notification for unknown message id "2881"
WARNING:pygls.protocol:Cancel notification for unknown message id "2880"
WARNING:pygls.protocol:Cancel notification for unknown message id "2904"
WARNING:pygls.protocol:Cancel notification for unknown message id "2913"
WARNING:pygls.protocol:Cancel notification for unknown message id "2915"
WARNING:pygls.protocol:Cancel notification for unknown message id "2916"
WARNING:pygls.protocol:Cancel notification for unknown message id "2933"
WARNING:pygls.protocol:Cancel notification for unknown message id "2940"
WARNING:pygls.protocol:Cancel notification for unknown message id "2951"
WARNING:pygls.protocol:Cancel notification for unknown message id "2953"
WARNING:pygls.protocol:Cancel notification for unknown message id "3011"
WARNING:pygls.protocol:Cancel notification for unknown message id "3031"
WARNING:pygls.protocol:Cancel notification for unknown message id "3034"
WARNING:pygls.protocol:Cancel notification for unknown message id "3048"
WARNING:pygls.protocol:Cancel notification for unknown message id "3049"
WARNING:pygls.protocol:Cancel notification for unknown message id "3050"
WARNING:pygls.protocol:Cancel notification for unknown message id "3058"
WARNING:pygls.protocol:Cancel notification for unknown message id "3059"
WARNING:pygls.protocol:Cancel notification for unknown message id "3061"
WARNING:pygls.protocol:Cancel notification for unknown message id "3068"
WARNING:pygls.protocol:Cancel notification for unknown message id "3070"
WARNING:pygls.protocol:Cancel notification for unknown message id "3071"
WARNING:pygls.protocol:Cancel notification for unknown message id "3075"
WARNING:pygls.protocol:Cancel notification for unknown message id "3085"
WARNING:pygls.protocol:Cancel notification for unknown message id "3087"
WARNING:pygls.protocol:Cancel notification for unknown message id "3089"
WARNING:pygls.protocol:Cancel notification for unknown message id "3090"
WARNING:pygls.protocol:Cancel notification for unknown message id "3091"
WARNING:pygls.protocol:Cancel notification for unknown message id "3086"
WARNING:pygls.protocol:Cancel notification for unknown message id "3093"
WARNING:pygls.protocol:Cancel notification for unknown message id "3092"
WARNING:pygls.protocol:Cancel notification for unknown message id "3088"
WARNING:pygls.protocol:Cancel notification for unknown message id "3096"
WARNING:pygls.protocol:Cancel notification for unknown message id "3094"
WARNING:pygls.protocol:Cancel notification for unknown message id "3095"
WARNING:pygls.protocol:Cancel notification for unknown message id "3098"
WARNING:pygls.protocol:Cancel notification for unknown message id "3099"
ERROR:pygls.protocol:Failed to handle request 3154 textDocument/codeAction partial_result_token=None work_done_token=None text_document=TextDocumentIdentifier(uri='file:///home/ubuntu/buenComensalProject/restaurants/api/api.py') range=8:18-8:18 context=CodeActionContext(diagnostics=[], only=None)
Traceback (most recent call last):
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request
    self._execute_request(msg_id, handler, params)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request
    method_name, method_type, msg_id, handler(params))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action
    extract_variable_refactoring = jedi_script.extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper
    return func(self, line, column, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable
    return extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable
    nodes = _find_nodes(module_node, pos, until_pos)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 68, in _find_nodes
    next_leaf = start_node.get_next_leaf()
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/parso/tree.py", line 121, in get_next_leaf
    i = c.index(node)
ValueError: PythonNode(trailer, [<Operator: (>, PythonNode(atom, [<Operator: [>, <String: 'GET'>, <Operator: ]>]), <Operator: )>]) is not in list
[Error - 3:41:20 AM] Request textDocument/codeAction failed.
  Message: ValueError: PythonNode(trailer, [<Operator: (>, PythonNode(atom, [<Operator: [>, <String: 'GET'>, <Operator: ]>]), <Operator: )>]) is not in list
  Code: -32602 
{'traceback': ['  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request\n    self._execute_request(msg_id, handler, params)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request\n    method_name, method_type, msg_id, handler(params))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action\n    extract_variable_refactoring = jedi_script.extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper\n    return func(self, line, column, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable\n    return extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable\n    nodes = _find_nodes(module_node, pos, until_pos)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 68, in _find_nodes\n    next_leaf = start_node.get_next_leaf()\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/parso/tree.py", line 121, in get_next_leaf\n    i = c.index(node)\n']}
WARNING:pygls.protocol:Cancel notification for unknown message id "3163"
ERROR:pygls.protocol:Failed to handle request 3187 textDocument/codeAction partial_result_token=None work_done_token=None text_document=TextDocumentIdentifier(uri='file:///home/ubuntu/buenComensalProject/restaurants/api/api.py') range=8:18-8:18 context=CodeActionContext(diagnostics=[], only=None)
Traceback (most recent call last):
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request
    self._execute_request(msg_id, handler, params)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request
    method_name, method_type, msg_id, handler(params))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action
    extract_variable_refactoring = jedi_script.extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper
    return func(self, line, column, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable
    return extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable
    nodes = _find_nodes(module_node, pos, until_pos)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 68, in _find_nodes
    next_leaf = start_node.get_next_leaf()
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/parso/tree.py", line 121, in get_next_leaf
    i = c.index(node)
ValueError: PythonNode(trailer, [<Operator: (>, PythonNode(atom, [<Operator: [>, <String: 'GET'>, <Operator: ]>]), <Operator: )>]) is not in list
[Error - 3:42:34 AM] Request textDocument/codeAction failed.
  Message: ValueError: PythonNode(trailer, [<Operator: (>, PythonNode(atom, [<Operator: [>, <String: 'GET'>, <Operator: ]>]), <Operator: )>]) is not in list
  Code: -32602 
{'traceback': ['  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request\n    self._execute_request(msg_id, handler, params)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request\n    method_name, method_type, msg_id, handler(params))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action\n    extract_variable_refactoring = jedi_script.extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper\n    return func(self, line, column, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable\n    return extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable\n    nodes = _find_nodes(module_node, pos, until_pos)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 68, in _find_nodes\n    next_leaf = start_node.get_next_leaf()\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/parso/tree.py", line 121, in get_next_leaf\n    i = c.index(node)\n']}
WARNING:pygls.protocol:Cancel notification for unknown message id "3194"
WARNING:pygls.protocol:Cancel notification for unknown message id "3193"
WARNING:pygls.protocol:Cancel notification for unknown message id "3195"
WARNING:pygls.protocol:Cancel notification for unknown message id "3197"
WARNING:pygls.protocol:Cancel notification for unknown message id "3198"
WARNING:pygls.protocol:Cancel notification for unknown message id "3201"
WARNING:pygls.protocol:Cancel notification for unknown message id "3208"
WARNING:pygls.protocol:Cancel notification for unknown message id "3209"
WARNING:pygls.protocol:Cancel notification for unknown message id "3213"
WARNING:pygls.protocol:Cancel notification for unknown message id "3215"
WARNING:pygls.protocol:Cancel notification for unknown message id "3216"
WARNING:pygls.protocol:Cancel notification for unknown message id "3222"
WARNING:pygls.protocol:Cancel notification for unknown message id "3225"
WARNING:pygls.protocol:Cancel notification for unknown message id "3227"
WARNING:pygls.protocol:Cancel notification for unknown message id "3248"
WARNING:pygls.protocol:Cancel notification for unknown message id "3284"
WARNING:pygls.protocol:Cancel notification for unknown message id "3288"
WARNING:pygls.protocol:Cancel notification for unknown message id "3321"
WARNING:pygls.protocol:Cancel notification for unknown message id "3353"
WARNING:pygls.protocol:Cancel notification for unknown message id "3354"
WARNING:pygls.protocol:Cancel notification for unknown message id "3361"
WARNING:pygls.protocol:Cancel notification for unknown message id "3381"
WARNING:pygls.protocol:Cancel notification for unknown message id "3391"
WARNING:pygls.protocol:Cancel notification for unknown message id "3402"
WARNING:pygls.protocol:Cancel notification for unknown message id "3489"
WARNING:pygls.protocol:Cancel notification for unknown message id "3488"
WARNING:pygls.protocol:Cancel notification for unknown message id "3491"
WARNING:pygls.protocol:Cancel notification for unknown message id "3490"
WARNING:pygls.protocol:Cancel notification for unknown message id "3493"
WARNING:pygls.protocol:Cancel notification for unknown message id "3500"
WARNING:pygls.protocol:Cancel notification for unknown message id "3501"
WARNING:pygls.protocol:Cancel notification for unknown message id "3502"
WARNING:pygls.protocol:Cancel notification for unknown message id "3506"
WARNING:pygls.protocol:Cancel notification for unknown message id "3507"
WARNING:pygls.protocol:Cancel notification for unknown message id "3512"
WARNING:pygls.protocol:Cancel notification for unknown message id "3518"
WARNING:pygls.protocol:Cancel notification for unknown message id "3520"
WARNING:pygls.protocol:Cancel notification for unknown message id "3526"
WARNING:pygls.protocol:Cancel notification for unknown message id "3538"
WARNING:pygls.protocol:Cancel notification for unknown message id "3539"
WARNING:pygls.protocol:Cancel notification for unknown message id "3541"
WARNING:pygls.protocol:Cancel notification for unknown message id "3583"
WARNING:pygls.protocol:Cancel notification for unknown message id "3602"
WARNING:pygls.protocol:Cancel notification for unknown message id "3629"
WARNING:pygls.protocol:Cancel notification for unknown message id "3649"
WARNING:pygls.protocol:Cancel notification for unknown message id "3651"
WARNING:pygls.protocol:Cancel notification for unknown message id "3653"
WARNING:pygls.protocol:Cancel notification for unknown message id "3654"
WARNING:pygls.protocol:Cancel notification for unknown message id "3650"
WARNING:pygls.protocol:Cancel notification for unknown message id "3652"
WARNING:pygls.protocol:Cancel notification for unknown message id "3655"
WARNING:pygls.protocol:Cancel notification for unknown message id "3656"
WARNING:pygls.protocol:Cancel notification for unknown message id "3660"
WARNING:pygls.protocol:Cancel notification for unknown message id "3705"
WARNING:pygls.protocol:Cancel notification for unknown message id "3727"
WARNING:pygls.protocol:Cancel notification for unknown message id "3729"
WARNING:pygls.protocol:Cancel notification for unknown message id "3732"
WARNING:pygls.protocol:Cancel notification for unknown message id "3746"
WARNING:pygls.protocol:Cancel notification for unknown message id "3748"
WARNING:pygls.protocol:Cancel notification for unknown message id "3782"
WARNING:pygls.protocol:Cancel notification for unknown message id "3787"
WARNING:pygls.protocol:Cancel notification for unknown message id "3785"
WARNING:pygls.protocol:Cancel notification for unknown message id "3817"
WARNING:pygls.protocol:Cancel notification for unknown message id "3820"
WARNING:pygls.protocol:Cancel notification for unknown message id "3822"
WARNING:pygls.protocol:Cancel notification for unknown message id "3823"
WARNING:pygls.protocol:Cancel notification for unknown message id "3827"
WARNING:pygls.protocol:Cancel notification for unknown message id "3828"
WARNING:pygls.protocol:Cancel notification for unknown message id "3829"
WARNING:pygls.protocol:Cancel notification for unknown message id "3837"
WARNING:pygls.protocol:Cancel notification for unknown message id "3850"
WARNING:pygls.protocol:Cancel notification for unknown message id "3920"
WARNING:pygls.protocol:Cancel notification for unknown message id "3930"
WARNING:pygls.protocol:Cancel notification for unknown message id "3935"
WARNING:pygls.protocol:Cancel notification for unknown message id "3959"
WARNING:pygls.protocol:Cancel notification for unknown message id "3977"
WARNING:pygls.protocol:Cancel notification for unknown message id "3979"
WARNING:pygls.protocol:Cancel notification for unknown message id "3982"
WARNING:pygls.protocol:Cancel notification for unknown message id "3990"
WARNING:pygls.protocol:Cancel notification for unknown message id "3993"
WARNING:pygls.protocol:Cancel notification for unknown message id "3991"
WARNING:pygls.protocol:Cancel notification for unknown message id "4006"
WARNING:pygls.protocol:Cancel notification for unknown message id "4007"
WARNING:pygls.protocol:Cancel notification for unknown message id "4011"
WARNING:pygls.protocol:Cancel notification for unknown message id "4012"
WARNING:pygls.protocol:Cancel notification for unknown message id "4014"
WARNING:pygls.protocol:Cancel notification for unknown message id "4022"
WARNING:pygls.protocol:Cancel notification for unknown message id "4027"
WARNING:pygls.protocol:Cancel notification for unknown message id "4029"
WARNING:pygls.protocol:Cancel notification for unknown message id "4042"
WARNING:pygls.protocol:Cancel notification for unknown message id "4051"
WARNING:pygls.protocol:Cancel notification for unknown message id "4060"
WARNING:pygls.protocol:Cancel notification for unknown message id "4064"
WARNING:pygls.protocol:Cancel notification for unknown message id "4065"
WARNING:pygls.protocol:Cancel notification for unknown message id "4067"
WARNING:pygls.protocol:Cancel notification for unknown message id "4068"
WARNING:pygls.protocol:Cancel notification for unknown message id "4070"
WARNING:pygls.protocol:Cancel notification for unknown message id "4066"
WARNING:pygls.protocol:Cancel notification for unknown message id "4069"
WARNING:pygls.protocol:Cancel notification for unknown message id "4071"
WARNING:pygls.protocol:Cancel notification for unknown message id "4072"
WARNING:pygls.protocol:Cancel notification for unknown message id "4073"
WARNING:pygls.protocol:Cancel notification for unknown message id "4088"
WARNING:pygls.protocol:Cancel notification for unknown message id "4099"
WARNING:pygls.protocol:Cancel notification for unknown message id "4106"
WARNING:pygls.protocol:Cancel notification for unknown message id "4110"
WARNING:pygls.protocol:Cancel notification for unknown message id "4109"
WARNING:pygls.protocol:Cancel notification for unknown message id "4111"
WARNING:pygls.protocol:Cancel notification for unknown message id "4170"
ERROR:pygls.protocol:Failed to handle request 4176 textDocument/codeAction partial_result_token=None work_done_token=None text_document=TextDocumentIdentifier(uri='file:///home/ubuntu/buenComensalProject/restaurants/api/api.py') range=24:56-24:56 context=CodeActionContext(diagnostics=[Diagnostic(range=24:55-24:56, message='SyntaxError: EOL while scanning string literal', severity=<DiagnosticSeverity.Error: 1>, code=None, code_description=None, source='jedi', related_information=None, tags=None, data=None)], only=None)
Traceback (most recent call last):
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request
    self._execute_request(msg_id, handler, params)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request
    method_name, method_type, msg_id, handler(params))
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action
    extract_variable_refactoring = jedi_script.extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper
    return func(self, line, column, *args, **kwargs)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable
    return extract_variable(
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable
    nodes = _find_nodes(module_node, pos, until_pos)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 87, in _find_nodes
    nodes = _remove_unwanted_expression_nodes(parent_node, pos, until_pos)
  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 196, in _remove_unwanted_expression_nodes
    nodes = nodes[start_index:end_index + 1]
UnboundLocalError: local variable 'end_index' referenced before assignment
[Error - 4:04:12 AM] Request textDocument/codeAction failed.
  Message: UnboundLocalError: local variable 'end_index' referenced before assignment
  Code: -32602 
{'traceback': ['  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 331, in _handle_request\n    self._execute_request(msg_id, handler, params)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/pygls/protocol.py", line 260, in _execute_request\n    method_name, method_type, msg_id, handler(params))\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi_language_server/server.py", line 551, in code_action\n    extract_variable_refactoring = jedi_script.extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/helpers.py", line 487, in wrapper\n    return func(self, line, column, *args, **kwargs)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/__init__.py", line 615, in extract_variable\n    return extract_variable(\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 19, in extract_variable\n    nodes = _find_nodes(module_node, pos, until_pos)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 87, in _find_nodes\n    nodes = _remove_unwanted_expression_nodes(parent_node, pos, until_pos)\n', '  File "/home/ubuntu/.vscode-server/extensions/ms-python.python-2022.16.0/pythonFiles/lib/jedilsp/jedi/api/refactoring/extract.py", line 196, in _remove_unwanted_expression_nodes\n    nodes = nodes[start_index:end_index + 1]\n']}
WARNING:pygls.protocol:Cancel notification for unknown message id "4203"
WARNING:pygls.protocol:Cancel notification for unknown message id "4219"
WARNING:pygls.protocol:Cancel notification for unknown message id "4233"
WARNING:pygls.protocol:Cancel notification for unknown message id "4234"
WARNING:pygls.protocol:Cancel notification for unknown message id "4236"
WARNING:pygls.protocol:Cancel notification for unknown message id "4261"
WARNING:pygls.protocol:Cancel notification for unknown message id "4263"
WARNING:pygls.protocol:Cancel notification for unknown message id "4264"
WARNING:pygls.protocol:Cancel notification for unknown message id "4266"
WARNING:pygls.protocol:Cancel notification for unknown message id "4265"
WARNING:pygls.protocol:Cancel notification for unknown message id "4269"
WARNING:pygls.protocol:Cancel notification for unknown message id "4270"
WARNING:pygls.protocol:Cancel notification for unknown message id "4273"
WARNING:pygls.protocol:Cancel notification for unknown message id "4272"
WARNING:pygls.protocol:Cancel notification for unknown message id "4274"
WARNING:pygls.protocol:Cancel notification for unknown message id "4276"
WARNING:pygls.protocol:Cancel notification for unknown message id "4284"
WARNING:pygls.protocol:Cancel notification for unknown message id "4285"
WARNING:pygls.protocol:Cancel notification for unknown message id "4286"
WARNING:pygls.protocol:Cancel notification for unknown message id "4287"
WARNING:pygls.protocol:Cancel notification for unknown message id "4288"
WARNING:pygls.protocol:Cancel notification for unknown message id "4289"
WARNING:pygls.protocol:Cancel notification for unknown message id "4290"
WARNING:pygls.protocol:Cancel notification for unknown message id "4291"
WARNING:pygls.protocol:Cancel notification for unknown message id "4292"
WARNING:pygls.protocol:Cancel notification for unknown message id "4293"
WARNING:pygls.protocol:Cancel notification for unknown message id "4294"
WARNING:pygls.protocol:Cancel notification for unknown message id "4304"
WARNING:pygls.protocol:Cancel notification for unknown message id "4305"
WARNING:pygls.protocol:Cancel notification for unknown message id "4306"
WARNING:pygls.protocol:Cancel notification for unknown message id "4308"
WARNING:pygls.protocol:Cancel notification for unknown message id "4339"
WARNING:pygls.protocol:Cancel notification for unknown message id "4351"
WARNING:pygls.protocol:Cancel notification for unknown message id "4370"
WARNING:pygls.protocol:Cancel notification for unknown message id "4373"
WARNING:pygls.protocol:Cancel notification for unknown message id "4377"
WARNING:pygls.protocol:Cancel notification for unknown message id "4383"
WARNING:pygls.protocol:Cancel notification for unknown message id "4384"
WARNING:pygls.protocol:Cancel notification for unknown message id "4388"
WARNING:pygls.protocol:Cancel notification for unknown message id "4389"
WARNING:pygls.protocol:Cancel notification for unknown message id "4390"
WARNING:pygls.protocol:Cancel notification for unknown message id "4391"
WARNING:pygls.protocol:Cancel notification for unknown message id "4393"
WARNING:pygls.protocol:Cancel notification for unknown message id "4394"
WARNING:pygls.protocol:Cancel notification for unknown message id "4396"
WARNING:pygls.protocol:Cancel notification for unknown message id "4397"
WARNING:pygls.protocol:Cancel notification for unknown message id "4392"
WARNING:pygls.protocol:Cancel notification for unknown message id "4398"
WARNING:pygls.protocol:Cancel notification for unknown message id "4395"
WARNING:pygls.protocol:Cancel notification for unknown message id "4400"
WARNING:pygls.protocol:Cancel notification for unknown message id "4411"
WARNING:pygls.protocol:Cancel notification for unknown message id "4430"
WARNING:pygls.protocol:Cancel notification for unknown message id "4451"
WARNING:pygls.protocol:Cancel notification for unknown message id "4455"
WARNING:pygls.protocol:Cancel notification for unknown message id "4454"
WARNING:pygls.protocol:Cancel notification for unknown message id "4470"
WARNING:pygls.protocol:Cancel notification for unknown message id "4479"
WARNING:pygls.protocol:Cancel notification for unknown message id "4491"
WARNING:pygls.protocol:Cancel notification for unknown message id "4492"
WARNING:pygls.protocol:Cancel notification for unknown message id "4493"
WARNING:pygls.protocol:Cancel notification for unknown message id "4495"
WARNING:pygls.protocol:Cancel notification for unknown message id "4494"
WARNING:pygls.protocol:Cancel notification for unknown message id "4539"
WARNING:pygls.protocol:Cancel notification for unknown message id "4573"
WARNING:pygls.protocol:Cancel notification for unknown message id "4607"
WARNING:pygls.protocol:Cancel notification for unknown message id "4613"
WARNING:pygls.protocol:Cancel notification for unknown message id "4616"
WARNING:pygls.protocol:Cancel notification for unknown message id "4617"
WARNING:pygls.protocol:Cancel notification for unknown message id "4618"
WARNING:pygls.protocol:Cancel notification for unknown message id "4659"
WARNING:pygls.protocol:Cancel notification for unknown message id "4662"
WARNING:pygls.protocol:Cancel notification for unknown message id "4672"
WARNING:pygls.protocol:Cancel notification for unknown message id "4680"
WARNING:pygls.protocol:Cancel notification for unknown message id "4685"
WARNING:pygls.protocol:Cancel notification for unknown message id "4701"
WARNING:pygls.protocol:Cancel notification for unknown message id "4732"
WARNING:pygls.protocol:Cancel notification for unknown message id "4738"
WARNING:pygls.protocol:Cancel notification for unknown message id "4742"
WARNING:pygls.protocol:Cancel notification for unknown message id "4743"
WARNING:pygls.protocol:Cancel notification for unknown message id "4748"
WARNING:pygls.protocol:Cancel notification for unknown message id "4749"
WARNING:pygls.protocol:Cancel notification for unknown message id "4752"
WARNING:pygls.protocol:Cancel notification for unknown message id "4753"
WARNING:pygls.protocol:Cancel notification for unknown message id "4754"
WARNING:pygls.protocol:Cancel notification for unknown message id "4757"
WARNING:pygls.protocol:Cancel notification for unknown message id "4761"
WARNING:pygls.protocol:Cancel notification for unknown message id "4780"
WARNING:pygls.protocol:Cancel notification for unknown message id "4779"
WARNING:pygls.protocol:Cancel notification for unknown message id "4783"
WARNING:pygls.protocol:Cancel notification for unknown message id "4782"
WARNING:pygls.protocol:Cancel notification for unknown message id "4781"
WARNING:pygls.protocol:Cancel notification for unknown message id "4791"
WARNING:pygls.protocol:Cancel notification for unknown message id "4793"
WARNING:pygls.protocol:Cancel notification for unknown message id "4794"
WARNING:pygls.protocol:Cancel notification for unknown message id "4790"
WARNING:pygls.protocol:Cancel notification for unknown message id "4797"
WARNING:pygls.protocol:Cancel notification for unknown message id "4792"
WARNING:pygls.protocol:Cancel notification for unknown message id "4798"
WARNING:pygls.protocol:Cancel notification for unknown message id "4796"
WARNING:pygls.protocol:Cancel notification for unknown message id "4795"
WARNING:pygls.protocol:Cancel notification for unknown message id "4799"
WARNING:pygls.protocol:Cancel notification for unknown message id "4800"
WARNING:pygls.protocol:Cancel notification for unknown message id "4801"
WARNING:pygls.protocol:Cancel notification for unknown message id "4802"
WARNING:pygls.protocol:Cancel notification for unknown message id "4812"
WARNING:pygls.protocol:Cancel notification for unknown message id "4814"
WARNING:pygls.protocol:Cancel notification for unknown message id "4815"
WARNING:pygls.protocol:Cancel notification for unknown message id "4817"
WARNING:pygls.protocol:Cancel notification for unknown message id "4838"
WARNING:pygls.protocol:Cancel notification for unknown message id "4882"
WARNING:pygls.protocol:Cancel notification for unknown message id "4885"
WARNING:pygls.protocol:Cancel notification for unknown message id "4889"
WARNING:pygls.protocol:Cancel notification for unknown message id "4892"
WARNING:pygls.protocol:Cancel notification for unknown message id "4898"
WARNING:pygls.protocol:Cancel notification for unknown message id "4920"
WARNING:pygls.protocol:Cancel notification for unknown message id "4922"
WARNING:pygls.protocol:Cancel notification for unknown message id "4932"
WARNING:pygls.protocol:Cancel notification for unknown message id "4986"
WARNING:pygls.protocol:Cancel notification for unknown message id "4988"
WARNING:pygls.protocol:Cancel notification for unknown message id "4999"
WARNING:pygls.protocol:Cancel notification for unknown message id "5004"
WARNING:pygls.protocol:Cancel notification for unknown message id "5011"
WARNING:pygls.protocol:Cancel notification for unknown message id "5023"
